supported-sports.json

```diff
@@ -1280,6 +1280,82 @@
                 { "type": "x", "title": "X", "id": "x", "noWidget": true },
                 { "type": "y", "title": "Y", "id": "y", "noWidget": true }
             ]
+        },
+        {
+            "id": "korfball",
+            "name": "Korfball",
+            "units": "meters",
+            "specifications": "IKF (2022)",
+            "dimensions": "40 x 20",
+            "scoringArea": "goal",
+            "perimeter": "#outside-perimeter",
+            "appearance": {
+                "width": "40",
+                "height": "20",
+                "circleR": "0.4",
+                "polyR": "0.55",
+                "fontSize": "0.03rem",
+                "strokeWidth": "0.075px",
+                "heatMapScale": 6.25,
+                "heatMapLowOpacity": 0.1,
+                "heatMapHighOpacity": 0.4
+            },
+            "goalCoords": [
+                [6.67, 10],
+                [33.33, 10]
+            ],
+            "defaultDetails": [
+                {
+                    "type": "shot-number",
+                    "title": "#",
+                    "id": "shot-number",
+                    "noWidget": true
+                },
+                {
+                    "type": "radio",
+                    "title": "Quarter",
+                    "id": "quarter",
+                    "options": [
+                        { "value": "1", "checked": true },
+                        { "value": "2" },
+                        { "value": "3" },
+                        { "value": "4" },
+                        { "value": "OT" }
+                    ]
+                },
+                {
+                    "type": "team",
+                    "title": "Team",
+                    "className": "team-select",
+                    "id": "team",
+                    "blueTeamName": "Home",
+                    "orangeTeamName": "Away",
+                    "checked": "blue-team-select"
+                },
+                {
+                    "type": "player",
+                    "title": "Player",
+                    "id": "player-input",
+                    "defaultValue": ""
+                },
+                {
+                    "type": "shot-type",
+                    "title": "Outcome",
+                    "id": "shot-type",
+                    "options": [
+                        { "value": "Miss", "selected": true },
```

Comment:
Done. I've reverted back to the same for basketball. I was wondering whether it would be better to make different teams the different shape, and different shot types a different colour? It might be more amenable to differentiating more shot types and there's typically only 2 teams?

Reply:
Ah yeah, this is definitely something that others have brought up in the past, and I've declined to change it for two main reasons. The first is convention and ease; teams are almost always associated with colors, in shot-plotter and other visualizations and in life (with team colors and such), and I generally want to stick to that convention to make it easier to parse. The second is practicality and accessibility: I've spent time making sure the orange and blue are as distinguishable as possible from the colors used in the playing areas (the one exception is monochromatic/greyscale, which I have on my list to tweak make more distinguishable), and there is definitely a hard limit of colors that are distinguishable from one another, especially in cases of color vision deficiencies. Shapes, on the other hand, there are much more possibilities; for example, I'm think of filling the first 10 or so options with shapes like stars and crosses and then switching back to the current regular polygon method afterward; there are many more reasonable and distinguishable options. With shot type, which I've technically left to possibly have as many options as possible, I want to accommodate that as best as can (even if currently trying to dodge it by default), which I can't do so well with colors.